### ISSUE TITLE
Monk causes constant desynchronisation in co-op

### ISSUE THREAD

**Issue by wealthbeyondmeasure** [SolastaMods/SolastaCommunityExpansion#1552]
Hi there,

I couldn't see anything about this in the issues list, so I wanted to report this to see if anyone had any insights or potential solutions.

My partner and I are playing a co op Lost Valley campaign. Our party consists of a Court Mage Wizard and a Scout Sentinel Artificer (under my partner's control) and a Marshal Fighter and an Open Hand Monk (under my control).

Since yesterday's update we have been experiencing constant desynchronisation issues specifically during the Monk's turn. On my screen I will do particular things, like attacking an enemy, but my partner will either see my character a) not doing anything at all or b) see completely different dice roll results. Once my turn ends we get stuck as my partner's screen is "waiting for me" and my screen is "waiting for partner". It's hard to pin down exactly when it happens but it seems to be something to do with the choice of using Stunning Strike on hit. Reloading doesn't seem to help much, usually giving a turn or two before the same issue happens again.

This only started happening since yesterday's patch (which we both had to install manually rather than updating in game like usual, if that matters) - prior to this point we had no issues in either modded or unmodded games.

Has anyone else experienced this or is there something we should alter in the settings?

Thanks for any support you can offer.

**Comment by wealthbeyondmeasure** [comment]
Hi,

Thanks very much for your response.

Please find the save attached. 

Reproduction is a bit inconsistent but generally:

1. Join a co-operative game hosted by somebody else
2. Enter a combat encounter
3. Play until it is the Monk's (Jadzia in this save) turn
4. Attack enemies with Jadzia's melee strikes
5. Choose whether or not to use the Stunning Strike reaction

Usually it is around this point that the desync occurs. Unfortunately it's hard to pinpoint the exact moment, but it's always on the Monk's turn.

[AutoSave_3.zip](https://github.com/SolastaMods/SolastaCommunityExpansion/files/9122387/AutoSave_3.zip)
.

**Comment by wealthbeyondmeasure** [comment]
Sure thing.

Here's the player.log from the MP host's machine:

[Player.log](https://github.com/SolastaMods/SolastaCommunityExpansion/files/9123689/Player.log)

And here's the player.log from my own machine:

[Player.log](https://github.com/SolastaMods/SolastaCommunityExpansion/files/9123692/Player.log)

**Comment by wealthbeyondmeasure** [comment]
To be honest after a few more failed gameplay sessions, I'm not sure it's the monk - we've had several issues where we rolled completely different initiatives at the start of combat and the like, so it didn't seem to be triggered by a particular event.

On doing some research I've seen that skill check boosters (like Bless or the Stone of Good Luck) can cause desync - and we have a Tinkerer in the party giving everyone Flash of Insight constantly, so that might be the issue.

Apologies if I've wasted your time. We're just frustrated by the desyncs and want to finish our campaign!
